README.md

```diff
@@ -162,6 +162,53 @@ spec:
       method: request.method | "get"
 ```
 
+#### OpenID Connect Pattern
+To use the *OpenID Connect authentication pattern*, you should use the `properties` value on the `subject` field to set `client_id`, and **optionally** `app_key`.
```

Comment:
@Darren - "should use" implies choice - since we're directing them out to use OpenID, should we change it to "use"?

Reply:
@seanhuck did you mean to ping @dfennessy by chance? Perhaps this comment was meant to be done in product documentation rather than here in the upstream? (btw, we welcome this kind of feedback as well!)